src/Frontend/src/stores/AuditStore.ts

```diff
@@ -27,6 +27,19 @@ export const useAuditStore = defineStore("AuditStore", () => {
   const selectedEndpointName = ref<string>("");
   const endpoints = ref<EndpointsView[]>([]);
 
+  const hasSuccessfulMessages = ref(false);
+
+  async function checkForSuccessfulMessages() {
+    try {
+      // Fetch the latest 10 messages and check if any are successful
+      // todo: ideally we would want to filter successful messages server-side, but the API doesn't currently support that
+      const [, data] = await serviceControlClient.fetchTypedFromServiceControl<Message[]>(`messages2/?page_size=10&sort=time_sent&direction=desc`);
```

Comment:
I wonder if you can sort by `status`, and successful messages would come on top?